script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -572,34 +610,38 @@ Oh, how sinful it was――how foolish.
 [sha:50053fbf301454207b49cf7174f1b19ac30f0abf]{
 -- Page 684, Offset 16501.
 -- 彼はこの時、宇宙のすべてを論理的に明かしたいと、天上の主に願ったのだ。
-It was then that he swore to the heavens above that he would uncover the theory of the entire cosmos.
+// Is 天上の主 supposed to be e.g. heavenly father? Or is it fine to be secular/generic here? Doesn't seem to be a common phrase
+It was then that he swore to the heavens above that he would lay bare the theory that underpins the cosmos.
 }
 [sha:0face707c32483762de03981afe6965e15ee588b]{
 -- Page 685, Offset 16502.
 -- だが、やがて彼は知る。
-But he would soon come to know...
+However, he was in for a rude awakening.
 }
 [sha:056e37f4148e891e141c7f634e1dd071fe924923]{
 -- Page 685, Offset 16503.
 -- 一生を費やすと決心した“あらまし”は、先人達によって解答にいきついていた事に。
-That the 'conceptualization' he had determined to spend his life uncovering, had already been achieved by his predecessors.
+The 'essence' that he had resolved to spend his life uncovering had already been deciphered by his predecessors.
 }
 [sha:6541cc8213e57a25c1dc542f58c02a98789e36cf]{
 -- Page 686, Offset 16504.
 -- 神の家は弾圧の名の下にあらゆる知識を貯蔵していた。
-// in the name of oppression is a strange sentence so...
-The house of God had amassed the knowledge of everything and all, through means of oppression.
+// Reading this more as suppression, as in the hoarding and concealment of knowledge, than oppression (of who?).
```

Comment:
I remember when I translated this that I saw it as them taking knowledge from the people they oppressed, but it's probably more in the censorship vein, yes.